### ISSUE TITLE
[bug] Input border styles with a theme like `blue` and  when focused on Native is not resolving to the colour defined inside the theme

### ISSUE THREAD

**Issue by sync** [tamagui/tamagui#358]
If you create a new component containing an `Input` and wrapped with a `<Theme>` or has a `theme` property set to for example to `blue`, the resolved `$borderColorFocus` ignore the theme and resolve for the default light theme value on `native` devices. 

Those styles here: https://github.com/tamagui/tamagui/blob/a701eb3f8c83f96d9ddce419388fdd710b8fa6c4/packages/tamagui/src/views/Input.tsx#L26

It is working fine on the web (see screenshot, the Input on iOS has a grey border when it should have a blue border based on the styles set from the `blue` theme). 

<img width="1367" alt="Screenshot 2022-11-13 at 6 22 55 pm" src="https://user-images.githubusercontent.com/21725/201510685-2baf747d-57e9-4cce-9775-3e9132d56140.png">

**To Reproduce**
Include steps to reproduce.

```
    <Theme name="blue">
      <YStack f={1} jc="center" ai="center" p="$4" space>
        <Input />

        <XStack>
          <Button {...linkProps}>Link to user</Button>
        </XStack>

      </YStack>
    </Theme>
```

Repo here: https://github.com/sync/tamago (run `yarn native:ios`)

**System Info (please complete the following information):** 

Output of `npx envinfo --system --npmPackages --binaries --browsers`

```
 System:
    OS: macOS 13.0.1
    CPU: (10) arm64 Apple M1 Pro
    Memory: 108.98 MB / 16.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 16.17.0 - ~/Library/Caches/fnm_multishells/67734_1668322708477/bin/node
    Yarn: 3.2.3 - ~/Library/Caches/fnm_multishells/67734_1668322708477/bin/yarn
    npm: 8.19.2 - ~/Library/Caches/fnm_multishells/67734_1668322708477/bin/npm
    Watchman: 2022.10.31.00 - /opt/homebrew/bin/watchman
  Browsers:
    Chrome: 107.0.5304.110
    Firefox Developer Edition: 104.0
    Safari: 16.1
  npmPackages:
    @babel/runtime: ^7.18.9 => 7.18.9 
    @manypkg/cli: ^0.19.1 => 0.19.1 
    @types/react: ^18.0.15 => 17.0.21 
    @types/react-native: ^0.69.5 => 0.69.5 
    eslint: ^8.21.0 => 8.21.0 
    prettier: ^2.7.1 => 2.7.1 
    turbo: ^1.4.2 => 1.4.2 
    typescript: ^4.7.4 => 4.7.4 
```

**Tamagui info**

Paste all the following below:

- [x] yarn.lock or similar
- [x] If it's a specific file, add `//! debug` to the top (the `!` works in esbuild compiled packages but not necessary always) and include that (only for compiler, but helps to know if it's on or off).
- [x] Run your server/bundler with `DEBUG=tamagui` and include that output at the bottom. (see the attached debug.txt file)

[debug.txt](https://github.com/tamagui/tamagui/files/9996659/debug.txt)

**Comment by sync** [comment]
How I could tell it was using the light theme `$borderColorFocus`  was by doing this.

<img width="636" alt="Screenshot 2022-11-14 at 9 27 23 am" src="https://user-images.githubusercontent.com/21725/201547771-3cb7a4b0-aada-4f03-914d-c42b32c90740.png">

**Comment by natew** [comment]
Can you try beta 215, there's some native style merge fixes.

**Comment by sync** [comment]
> 

same issue sorry. Pushed the changes to the sample repo here:  https://github.com/sync/tamago

**Comment by natew** [comment]
Good catch, fixed in b29bc4a6be9c0a67612364a569b995d9c5f58454 just waiting for CI to releas

**Comment by sync** [comment]
tested, I can confirm the issue was fixed in https://github.com/tamagui/tamagui/commit/b29bc4a6be9c0a67612364a569b995d9c5f58454 (1.0.1-beta.216),  thank you 🍭
